src/cloudinary-image.component.spec.ts

```diff
@@ -101,6 +101,7 @@ describe('CloudinaryImage', () => {
             <cl-transformation width="300" crop="scale" overlay="text:roboto_35_bold:SDK"></cl-transformation>
             <cl-transformation effect="art:hokusai"></cl-transformation>
             <cl-transformation fetch-format="auto"></cl-transformation>
+            <cl-transformation if="initialWidth > 400"></cl-transformation>
```

Comment:
A complete example must have an "if_end" clause in it, otherwise the generated URL is invalid.

Please change to 

```
            <cl-transformation if="initialWidth > 400"></cl-transformation>
            <cl-transformation angle="10"></cl-transformation>
            <cl-transformation if="end"></cl-transformation>
```

And updated the expectations accordingly

Reply:
Ok, will update it, but we need an example of a valid URL to validate it